### PR TITLE
Chat component

### PR DESCRIPTION
**Pull request by betulerkoc**
- chat component added with all functionality

chatpng

### CODE REVIEW

**Review by knargocer — APPROVED** [review]

File: src/components/Chat/index.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
+main {
```
Comment (Ammar-64) [inline src/components/Chat/index.css]: Can you give classNames to this css file, this way it’ll affect all the elements on the website

File: src/components/Chat/index.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  );
+}
+
+function ChatMessage(props) {
+  const { text, uid, photoURL } = props.message;
+  const messageClass = uid === auth.currentUser.uid ? "sent" : "received";
+  return (
+    <>
+      <div className={`message ${messageClass}`}>
+        <img alt="" src={photoURL} />
```
Comment (Ammar-64) [inline src/components/Chat/index.jsx]: An img should has a text alt

File: src/components/Chat/index.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      <header>
+        <SignOut />
+      </header>
+
+      <section>{user ? <ChatRoom /> : <SignIn />}</section>
+    </div>
+  );
+}
+
+function SignIn() {
```
Comment (Ammar-64) [inline src/components/Chat/index.jsx:25]: I think you can import and use the sign in and sign out created by orcun before instead of doing them again

**Review by Ammar-64 — CHANGES_REQUESTED** [review]
Solid but please take these comments into consideration

File: src/components/Chat/index.jsx (hunk 3 above)
Comment (betulerkoc) [reply src/components/Chat/index.jsx:25]: The current sign in doesn't work properly, that's why i need to add this. After this is merged, will try to take this outside and use everywhere

File: src/components/Chat/index.jsx (hunk 2 above)
Comment (betulerkoc) [reply src/components/Chat/index.jsx]: PTAL

File: src/components/Chat/index.css (hunk 1 above)
Comment (betulerkoc) [reply src/components/Chat/index.css]: PTAL

**Review by Ammar-64 — APPROVED** [review]
LGTM

**Pull request merged**